src/flag.rs

```diff
@@ -30,6 +31,7 @@ pub enum FlagValue {
     Bool(bool),
     String(String),
     Int(isize),
+    U64(u64),
```

Comment:
```suggestion
    Uint(u64),
```
Rename it because the name is inconsistent with `Int(isize)`

Reply:
Done